### DISCUSSION TITLE
Has anyone successfully created a custom audit policy with v0.16.1?

### DISCUSSION THREAD

**Discussion by data-dude** [aquasecurity/trivy#5734] [Q&A]
### Question

I follow the below directions and Trivy doesn't pick up my custom audit policy:
https://aquasecurity.github.io/trivy-operator/v0.16.1/tutorials/writing-custom-configuration-audit-policies/
I also created an extremely simple policy that sets a value to true and when kubectl apply the policy trivy rescans all the pods but for whatever reason it doesn't pick up my custom audit.  Is there some trick that I'm missing?

### Target

Kubernetes

### Scanner

Misconfiguration

### Output Format

JSON

### Mode

Standalone

### Version

```bash
v0.16.1
```

**Comment by knqyf263** [comment]
You can raise it [here](https://github.com/aquasecurity/trivy-operator)
